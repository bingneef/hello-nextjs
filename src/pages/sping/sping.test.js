import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";

import Sping from "./sping";

describe("Sping", () => {
  it("renders without error", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Sping />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("matches the snapshot", () => {
    const component = renderer.create(<Sping title="Sping" />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    const rootInstance = component.root;
    const div = rootInstance.findByType("div");
    expect(div.props.children).toBe("Sping");
  });
});
