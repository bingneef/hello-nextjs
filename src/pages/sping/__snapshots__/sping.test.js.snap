// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sping matches the snapshot 1`] = `
<div
  className="root"
>
  Sping
</div>
`;
